openstates/mn/__init__.py

```diff
@@ -1,10 +1,11 @@
-import lxml.html
-import gc
+# encoding=utf-8
```

Comment:
This is the default source code encoding in python3, so I don't think we need it.